account/lnurl.go

```diff
@@ -36,6 +36,13 @@ type LoginResponse struct {
 func (a *Service) HandleLNURL(rawString string) (result *data.LNUrlResponse, err error) {
 	var lightningAddress string
 
+	if tor := a.TorConfig; tor != nil {
```

Comment:
Isn't it better to initialize the TorClient on startup when we detect that tor is enabled? We have a single point when we initialize the tor config and it makes sense to keep it this way.

Reply:
Good point. Enabling it on startup now.